test/unit/org/apache/cassandra/index/sai/iterators/KeyRangeConcatIteratorTest.java

```diff
@@ -33,8 +33,8 @@
 public class KeyRangeConcatIteratorTest extends AbstractKeyRangeIteratorTester
 {
     private static final String MUST_BE_SORTED_ERROR = "RangeIterator must be sorted, previous max: PrimaryKey: " +
-                                                       "{ token: %d, partition: null, clustering: null:null} , " +
-                                                       "next min: PrimaryKey: { token: %d, partition: null, clustering: null:null} ";
+                                                       "{ token: %d }, " +
+                                                       "next min: PrimaryKey: { token: %d }";
```

Comment:
nit: Not terribly important, but could create a constant in `KeyRangeConcatIterator` and reuse here...modulo the `PrimaryKey` bits...